pkg/routeagent_driver/handlers/mtu/mtuhandler.go

```diff
@@ -24,6 +24,11 @@ import (
 	"github.com/submariner-io/submariner/pkg/netlink"
 )
 
+const (
+	mtuProbe = "2"
```

Comment:
nit: A comment about what "2" means would help users understand what we are trying to achieve.

Reply:
Done.